list_all_versions_offline

```diff
@@ -1,3 +1,78 @@
+0.48.0
+0.47.0
+0.46.3
```

Comment:
can you check the history of that file - and just validate that we didn't remove some versions on purpose 🤔 
versions with security issue / bug 🤔

Reply:
Checked, no version was removed.